flexget/tests/conftest.py

```diff
@@ -237,7 +238,7 @@ def filecopy(request):
                 else:
                     f.remove()
             except OSError as e:
-                print("couldn't remove %s: %s" % (f, e))
+                print ("couldn't remove %s: %s" % (f, e))
```

Comment:
added space by mistake?

Reply:
Hmm, it happened twice in that commit, I remember seeing it in the diff after I ran black, but assumed it was fixing the extra space, not adding one...

Reply:
Ahh, I must need to upgrade black on that machine. https://github.com/psf/black/issues/752